app/screens/initial/InitScreen.tsx

```diff
@@ -37,7 +39,58 @@ const InitScreen: React.FC<InitScreenProp> = ({ navigation }) => {
           />
         </Center>
         <VStack space={2}>
-          <Button variant="solid" onPress={() => navigation.navigate('Login')}>
+          <Button
+            variant="filledGreen"
```

Comment:
지금도 동작은 잘 하는데 `variant`의 정의를 유지한다면 아래와 같이 작성할 수 있을 것 같아요! 이렇게 작성하고 `theme.ts`의 `filledGreen`를 삭제하면 테마를 따로 추가하지 않고 기존 설정만으로 커스텀할 수 있습니다.
```suggestion
            variant="solid"
            bgColor="#2DB400"
```

Reply:
[d3afd28](https://github.com/mountain-sailors/stock-battle-front/pull/65/commits/d3afd287b58fc0a04157837ae22175a25d074f53) 에 수정완료했습니다~